Test/dafny0/RefinementErrors.dfy

```diff
@@ -183,10 +183,10 @@ module OrigJ {
   }
 }
 module TpJ refines OrigJ {
-  iterator S<A,B>()
-  // Note, type parameters of iterators are implicitly (0), so
-  // the following line generates 7, not 9, errors
-  iterator T<!A,-B,+C,!D,-E,+F,W(==),X,Y(0),Z>()  // error (x7): various changes
-  {
+  iterator S ...
```

Comment:
So, type parameters are never allowed to be repeated?

Reply:
That was the original grammar. We can discuss what flexibility to allow (it will take some implementation)